#Research

The Thornton lab does research in several areas of genomics, population genetics, and statistical genetics. In general, we are interested in using genomic data to understand how genomes evolve. Projects in my lab typically involve large data sets and/or extensive computer simulation. In either case, day-to-day work in the lab tends to emphasize using computational methods to understand something about genetics and evolution.

The primary research organisms are fruit flies from various species related to _Drosophila melanogaster_. Recently I have expanded into modeling human association studies in light of explicit population-genetic models.

Please see the [Publications](pubs.html) page for what the lab is up to.

If any of these projects sound interesting to you, please visit our page on joining the Thornton lab.

Current projects in the lab include:

* The population genomics of _Drosophila simulans_ and _Drosophila yakuba_, two close relatives of _D. melanogaster_. In collaboration with Peter Andolfatto’s lab at Princeton, we have generated the complete genome sequences of 20 individuals from each species, and have re-sequenced the reference strains for each, for a total of 42 genomes. The Thornton lab at UC Irvine prepared the sequencing libraries which were then sequenced at the UCI Genomics High Throughput Core Facility. The minimum sequencing coverage of a single genome is over 50x, and the Thornton lab is currently using these data to describe the extent of copy-number variation in these species and variation due to transposable-element insertions. Related publications include Cridland and Thornton (2010) Genome Biology and Evolution, Mackay et al. (2012) Nature, and Hu et al. (2012) Genome Research.
* Gene expression and structural variation in Drosophila. We have collected gene expression data using messenger-RNA sequencing (“RNA-seq”) from female flies from the majority of our D. simulans and D. yakuba stocks for which we have complete genomes. Ongoing work in the lab involves using these data to ask if duplication really does lead to more gene product (at the level of transcription), the extent to which there are null alleles segregating due to large deletions, transposable element insertions, etc.
* Population-genetic models of disease risk. Recent efforts to identify the mutations in the human population which lead to increased risk of genetic disorders are currently able to account for only a small fraction of heritable risk. This “missing heritability” in human association studies has lead to speculation that rare alleles (currently not well-assayed in association studies) must be where the missing heritability is hiding. The literature attempting to model the properties of rare risk alleles typically avoids explicit population-genetic modeling principles, perhaps because they are computationally-challenging. As a result, it is unclear how to efficiently test for an effect of rare alleles in an association study, and it is also unclear what the population-genetic properties of genes harboring risk alleles are. A first, and small, step, at addressing some of these questions was published in Thornton, Foran, and Long (2013), PLoS Genetics. Extending the modeling done in that paper is fertile ground for future theoretical/statistical work on human association studies.
* The modeling of quantitative traits adapting to new environments
* The development of efficient software tools for population-genetic simulation

The lab has active and ongoing collaborations with:

* [Peter Andolfatto](http://genomics.princeton.edu/AndolfattoLab/Andolfatto_Lab.html), Princeton University 
* [Tony Long](xhttp://wfitch.bio.uci.edu/~tdlong/), UC Irvine