<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta http-equiv="Content-Style-Type" content="text/css" />
  <meta name="generator" content="pandoc" />
  <title></title>
  <style type="text/css">code{white-space: pre;}</style>
  <link rel="stylesheet" href="../pandoc.css" type="text/css" />
  <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
  <html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta http-equiv="Content-Style-Type" content="text/css" />
    <meta name="generator" content="pandoc" />
    <title></title>
    <style type="text/css">code{white-space: pre;}</style>
    <link rel="stylesheet" href="pandoc.css" type="text/css" />
  </head>
  <body>
  <p><a href="http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1003258"> <img src="http://molpopgen.org/images/journal.pgen.1003258.g001.png" style="width: 300px;"></a> <a href="http://mbe.oxfordjournals.org/content/31/7/1750.full"> <img src="http://www.molpopgen.org/images/F6.large.jpg" style="width: 300px;" height="300px"></a> <a href="http://mbe.oxfordjournals.org/content/30/10/2311.full"> <img src="http://www.molpopgen.org/images/F4.large.jpg" style="width: 300px;" height="300px"></a></p>
  <h4 id="home-publications-research-people-news-software-data-resources-blog"><a href="index.html">Home</a> || <a href="http://www.molpopgen.org/pubs.html">Publications</a> || <a href="http://www.molpopgen.org/research.html">Research</a> || <a href="http://www.molpopgen.org/people.html">People</a> || <a href="http://www.molpopgen.org/news.html">News</a> || <a href="http://www.molpopgen.org/software.html">Software</a> || <a href="http://www.molpopgen.org/data.html">Data</a> || <a href="http://www.molpopgen.org/res.html">Resources</a> || <a href="http://www.molpopgen.org/blog.html">Blog</a></h4>
  </body>
  </html>
</head>
<body>
<h1 id="reading-binary-data-in-r">Reading binary data in R</h1>
<p>In <a href="http://www.r-project.org">R</a>, you can read binary data using the <a href="https://stat.ethz.ch/R-manual/R-devel/library/base/html/readBin.html">readBin</a> function and write it with <a href="https://stat.ethz.ch/R-manual/R-devel/library/base/html/readBin.html">writeBin</a>.</p>
<p>In a previous <a href="ms2bin.html">post</a>, I showed how to convert data from the coalescent simulation program <a href="http://http://home.uchicago.edu/~rhudson1/source/mksamples.html">ms</a> to a more compact compressed format.</p>
<p>The following R code shows how to use readBin to read in the first record of such data:</p>
<script src="https://gist.github.com/molpopgen/9e8f7ab2711ab751420e.js"></script>
<h2 id="why-do-this">Why do this?</h2>
<p>Reading/writing binary is typically a lot faster than with plain-text data. Also, you don’t lose any precision due to rounding floating-point values, which can be handy. For R, I think that reading data in this way should be reserved for cases where data frames, etc., are not the best way to represent data. In the above, example, the 0/1 matrix is great for calculating summary statistics, etc. However, a lot of your data for R may be better-served by being in a “tidy” format for processing with packages like <a href="http://cran.r-project.org/web/packages/dplyr/index.html">dplyr</a>. Further, the new <a href="http://cran.r-project.org/web/packages/data.table/index.html">data.table</a> package reads in files very quickly, and into a data frame-like object.</p>
<p>Outside of R, there are lots of applications for binary files. <a href="http://samtools.github.io/hts-specs/SAMv1.pdf">Bam</a> files are one obvious example, but we find it a very useful way to store the output of simulation, and I imagine that future bioinformatics projects will use binary formats, too.</p>
</body>
</html>
