##Data

* [Alignments](http://devlaeminck.bio.uci.edu/Data/FastX) from Thornton, Bachtrog, and Andolfatto (2006) X-chromosomes and autosomes evolve at similar rates in Drosophila - no evidence for faster-X protein evolution. Genome Research 16: 498-504 

* [Alignments and annotations](http://devlaeminck.bio.uci.edu/Data/ThorntonLong2005MBE) from Thornton and Long (2005) Excess of Amino Acid Substitutions Relative to Polymorphism Between X-Linked Duplications in Drosophila melanogaster. Mol. Biol. Evol. 22: 273-284. This contains the alignments used for summary statistics, and the annotation files compatible with "polydNdS" in my analysis software package. It currently does not contain the files used for McDonald-Kreitman tests, or the analysis of gene conversion between paralogs. For the moment, those are available on request to Kevin Thornton.

* [Simulation source code and output](http://devlaeminck.bio.uci.edu/ThorntonForanLongPLoSGenetics.html) from Thornton, Foran, and Long (2013) Properties and Modeling of GWAS when Complex Disease Risk is due to non-complementing, deleterious mutations in genes of large effect. PLoS Genetics 9: e1003258

* [Source code and other resources](http:/devlaeminck.bio.uci.edu/tepipeline/) from Cridland, MacDonald, Long, and Thornton (2013) Abundance and Distribution of Transposable Elements in two Drosophila QTL Mapping Resources. Molecular Biology and Evolution. 

* A [version](http://devlaeminck.bio.uci.edu/tepipeline/line99_example.tar.gz) of the Cridland et al. (2013) pipeline for TE detection with example data is also available. This version comes with simulated sample data and a master script to execute the pipeline from start to finish.

* [Code](https://github.com/molpopgen/baldwin_brown_2014) from Baldwin-Brown, Long, and Thornton (2014) Mol Biol Evol is on Thornton’s github repository.

* Supplementary material from Rogers et al. (submitted) Landscape of tandem duplication in Drosophila yakuba and Drosophila simulans. Please note: the BED files for D. simulans are based on the w501 reference genome, hosted at Peter Andolfatto’s website. If you plug our BED files into the UCSC browser, you’ll be quite a bit off in terms of coordinates, as they are still using the Begun et al. reference.